### PR TITLE
fix: assume working dir can change at run time

### PR DESCRIPTION
**Pull request by creisle**
- Set node_path and use absolute path for entrypoint since singularity
  will not respect the working dir statement from docker
- Add gkb config args to snakemake

### CODE REVIEW

**Comment by mathieulemieux** [comment]
Is the Singularity thing related to this issue : https://stackoverflow.com/questions/65642199/difference-between-working-directory-of-docker-and-singularity ? Will you then use the NODE_PATH environment variable in command line when using Singularity?

**Comment by creisle** [comment]
Yup, that's why I added NODE_PATH to the docker definition. We won't need to add it in the command line though. it seems to do the trick as is since this sets the default value of that ENV at runtime

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,10 @@
...
 WORKDIR /usr/src/app
+# Bundle app source
 COPY package*.json ./
 RUN npm ci --only=production
-# Bundle app source
+# COPY everything not in dockerignore file
 COPY . .
-ENTRYPOINT [ "node", "bin/load.js" ]
+# set to avoid errors when singularity overloads working dir
+ENV NODE_PATH=/usr/src/app/node_modules
```
Comment (elewis2) [inline Dockerfile:9]: is the absolute path here going to be a problem if a user's path is different?

File: README.md [hunk 2]
```diff
@@ -168,6 +167,15 @@ snakemake -j 1
...
+```bash
+snakemake -j 1 \
+  --config gkb_user='graphkb_importer' \
+  gkb_pass='secret' \
+  gkb_url='http://localhost:8080/api'
+```
+
 The COSMIC and DrugBank options require licensing and are therefore not run by default. If you have
 a license to use them then you can include one or both of them by providing email and password
 as config parameters
```
Comment (elewis2) [inline README.md:181]: it would be snakefile -h to check the config parameter names?

Comment (creisle) [reply README.md:181]: i don't know that there is a way to do that actually. snakemake -h would just give you the snakemake help menu

File: Dockerfile (hunk 1 above)
Comment (creisle) [reply Dockerfile:9]: no b/c this is the path inside the docker container

**Review by mathieulemieux — APPROVED** [review]
Everything seems fine to me. At some point, I'd like if someone could walk me through these workflow diagrams... ;-)

**Review by elewis2 — APPROVED** [review]

**Pull request merged**